homeassistant/components/lutron_caseta.py

```diff
@@ -41,9 +41,7 @@ def setup(hass, base_config):
 
     config = base_config.get(DOMAIN)
     hass.data[LUTRON_CASETA_SMARTBRIDGE] = Smartbridge(
-        hostname=config[CONF_HOST],
-        username=config[CONF_USERNAME],
-        password=config[CONF_PASSWORD]
```

Comment:
Optional username and password suggest that users still can provide them if the setup requires it. This change looks like that authentication with username/password is no longer supported.

If so, this would become a breaking change.